src/Microsoft.TemplateEngine.Abstractions/ILocalizationModel.cs

```diff
@@ -20,6 +20,9 @@ public interface ILocalizationModel
         // symbol name -> localized description
         IReadOnlyDictionary<string, string> SymbolDescriptions { get; }
 
+        // symbol name -> choice -> localized description
+        IReadOnlyDictionary<string, IReadOnlyDictionary<string, string>> ChoiceDescriptions { get; }
```

Comment:
It feels like we should have a model type here for the symbols which includes the description & a breakdown of the choices vs descriptions internally instead of two top level properties that index by symbol